okhttp/src/main/kotlin/okhttp3/internal/tls/BasicCertificateChainCleaner.kt

```diff
@@ -75,7 +75,7 @@ class BasicCertificateChainCleaner(
       val i = queue.iterator()
       while (i.hasNext()) {
         val signingCert = i.next() as X509Certificate
-        if (verifySignature(toVerify, signingCert)) {
+        if (verifySignature(toVerify, signingCert, result.size - 1)) {
```

Comment:
Maybe worth a comment on these values. -1 is for the site certificate but not ignoring the trusted root.  Does this work when we found a trusted certificate above, but it wasn't accepted in verifySignature?